src/servers/src/error.rs

```diff
@@ -245,6 +245,13 @@ pub enum Error {
 
     #[snafu(display("{}", reason))]
     UnexpectedResult { reason: String, location: Location },
+
+    #[allow(dead_code)]
+    #[snafu(display("Servers internal error, source: {}", source))]
+    ServersInternal {
```

Comment:
How about just naming it as `Internal` and remove `Servers` from its display. The error message may be displayed to end user and `Servers` is not a meaningful name.

Reply:
A lot of crates like `Servers` has its own [`Internal` error](https://github.com/greptimeTeam/greptimedb/blob/0c1dc3313f04780ad3deb600e52b5a188219646a/src/servers/src/error.rs#L38) which merely prints a msg. Here we want a boxed error instead.